pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -725,6 +726,22 @@ func (adc *attachDetachController) processVolumeAttachments() error {
 				err)
 			continue
 		}
+		inTreePluginName := plugin.GetPluginName()
```

Comment:
nit - the name of variable  `inTreePluginName` is not correct. It could be CSI or in-tree plugin name depending on PV being linked to VA object.

Reply:
there is also - `adc.csiMigratedPluginManager.IsMigratable(volumeSpec)` if you want to use.

Reply:
Good point! Iet's change the var name to `pluginName` for now. And I think IsMigrationEnabledForPlugin works just fine here.

Reply:
I think this code need to use IsMigratable(). I think the volumeSpec here could already CSI. I think it may not be safe to call TranslateInTreeSpecToCSI() on CSI volumes.

Reply:
If it is already CSI, then `IsMigrationEnabledForPlugin` will return false and it will not translate the spec so I think it should be okay?